sklearn/feature_selection/_rfe.py

```diff
@@ -646,7 +653,13 @@ class RFECV(RFE):
 
         If `callable`, overrides the default feature importance getter.
         The callable is passed with the fitted estimator and it should
-        return importance for each feature.
+        return importance for each feature. When it accepts it, the callable is passed
+        `feature_indices` which stores the index of the features in the full dataset
+        that have not been eliminated yet.
+
+        `feature_indices` allows `RFECV` to be used with permutation importance, as
+        shown at the end of
+        :ref:`sphx_glr_auto_examples_feature_selection_plot_rfe_with_cross_validation.py`.
 
         .. versionadded:: 0.24
```

Comment:
I think we should add a `.. versionchanged:: 1.8` and mention that support for passing `feature_indices` to the callable when part of its signature.